src/distributions/uvnormal.jl

```diff
@@ -0,0 +1,14 @@
+# add docstr
+function Normal(μ::Number, Σ::Number)
+    T = promote_type(typeof(μ), typeof(Σ))
+    return Normal{T}(convert(T, μ), convert(T, Σ))
```

Comment:
Why the convesion actually? Wouldn't it be more convenient to keep the types as they are? Or does that create some weird unforseen errors?

Reply:
That is one concern I have. Another is that every operation involving both mean and cov would have to make a conversion anyway, so it might be more performant to simply enforce a uniform number type?

Reply:
Which concern is the one you have? I don't quite see what could go wrong here which is why I'm asking

Reply:
The unforseen errrors bit.